dali/pipeline/graph/graph_descr.cc

```diff
@@ -55,17 +56,17 @@ void CheckOpConstraints(const OpSpec &spec) {
   const int additional_outputs = schema.CalculateAdditionalOutputs(spec);
 
   DALI_ENFORCE(schema.SupportsInPlace(spec) || !spec.GetArgument<bool>("inplace"),
-      "Op '" + spec.SchemaName() + "' does not support in-place execution.");
+      "Operator `" + GetOpDisplayName(spec, true) + "` does not support in-place execution.");
   DALI_ENFORCE(spec.NumRegularInput() <= schema.MaxNumInput(),
-      "Operator '" + spec.SchemaName() +
-      "' supports a maximum of " + std::to_string(schema.MaxNumInput()) + " inputs, "
+      "Operator `" + GetOpDisplayName(spec, true) +
+      "` supports a maximum of " + std::to_string(schema.MaxNumInput()) + " inputs, "
       "but was passed " + std::to_string(spec.NumRegularInput()) + ".");
   DALI_ENFORCE(spec.NumRegularInput() >= schema.MinNumInput(),
-      "Operator '" + spec.SchemaName() +
-      "' supports a minimum of " + std::to_string(schema.MinNumInput()) + " inputs, "
+      "Operator `" + GetOpDisplayName(spec, true) +
+      "` supports a minimum of " + std::to_string(schema.MinNumInput()) + " inputs, "
       "but was passed " + std::to_string(spec.NumRegularInput()) + ".");
   DALI_ENFORCE(spec.NumOutput() == schema.CalculateOutputs(spec) + additional_outputs,
-      "Operator '" + spec.SchemaName() + "' supports "
+      "Operator `" + GetOpDisplayName(spec, true) + "` supports "
```

Comment:
Upgrade to make_string? You'll be able to drop all those `std::to_string`...

Reply:
done